notebooks/002-openvino-api/002-openvino-api.ipynb

```diff
@@ -126,32 +126,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from openvino.runtime import Core\n",
+    "from openvino.runtime import serialize, Core\n",
     "\n",
     "ie = Core()\n",
-    "onnx_model_path = \"model/segmentation.onnx\"\n",
-    "model_onnx = ie.read_model(model=onnx_model_path)\n",
-    "compiled_model_onnx = ie.compile_model(model=model_onnx, device_name=\"CPU\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "06f4a187",
-   "metadata": {},
-   "source": [
-    "The ONNX model can be exported to OpenVINO IR with `.serialize()`:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "a4fb397f",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from openvino.offline_transformations import serialize\n",
-    "\n",
-    "serialize(model=model_onnx, model_path=\"model/exported_onnx_model.xml\", weights_path=\"model/exported_onnx_model.bin\")"
+    "model = ie.read_model(\"model/segmentation.onnx\")\n",
+    "serialize(model, \"model/exported_onnx_model.xml\")"
```

Comment:
According to the contribution guide, all arguments should have corresponding keywords.